OpenRA.Mods.Common/Traits/BotModules/BotModuleLogic/BaseBuilderQueueManager.cs

```diff
@@ -147,7 +147,9 @@ bool TickQueue(IBot bot, ProductionQueue queue)
 					var possibleBuilding = world.ActorsWithTrait<Pluggable>().FirstOrDefault(a =>
 						a.Actor.Owner == player && a.Trait.AcceptsPlug(a.Actor, plugInfo.Type));
 
+#pragma warning disable CS8073 // The result of the expression is always the same since a value of this type is never equal to 'null'
```

Comment:
This smells like a real bug. Shouldn't we be cehcking `possibleBuilding.Actor` here instead?

Reply:
Cant build without this,

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS8073	The result of the expression is always 'true' since a value of type 'TraitPair<Pluggable>' is never equal to 'null' of type 'TraitPair<Pluggable>?'	OpenRA.Mods.Common	C:\projects\OpenRA\OpenRA.Mods.Common\Traits\BotModules\BotModuleLogic\BaseBuilderQueueManager.cs	150	Active

```